CIScripts/Test/PesterLogger/PesterLogger.Tests.ps1

```diff
@@ -0,0 +1,137 @@
+﻿$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path) -replace '\.Tests\.', '.'
+. "$here\$sut"
+
+function ItBlockOutsideOfDescribe() {
+    It "getting Pester scope works in It block outside of Describe" {
+        Get-CurrentPesterScope | Should -BeExactly @("PesterLogger", "It blocks in weird places",
+            "getting Pester scope works in It block outside of Describe")
+    }
+}
+
+Describe "PesterLogger" {
+    Context "when inside Context block" {
```

Comment:
I'd put all the tests of `Get-CurrentPesterScope` in a separate `Context`, or perhaps even `Describe`. The tests should serve as examples, but now the reader is directly greeted with testing implementation details of `Get-CurrentPesterScope`